src/s2wasm.h

```diff
@@ -372,8 +372,22 @@ class S2WasmBuilder {
 
   void prepare() {
     assert(nextStatic == globalBase); // we are the first allocation
+    // Allocate space for a user stack
+    staticAddresses[".stack"] = nextStatic;
```

Comment:
I think we plan to allow catchable faults on arbitrary ranges, so we could detect such overflows elsewhere as well. At least that's my recollection here, I could be wrong.

In any case the code size different is significant. I don't remember numbers in emscripten, but we switched to this after noticing it was a worthwhile code size win (on JS numbers, but LEB128 would be sort of similar).